rpc_jobs/rpc_aio.yml

```diff
@@ -169,6 +169,7 @@
 
     dsl: |
       // CIT Slave node
+      currentBuild.result = 'SUCCESS'
```

Comment:
Failures in delPubCloudSlave should be caught by the catch on line 227, so that shouldn't cause an inconsistency, whats the situation you are concerned about?